.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: java
+jdk:
+- openjdk11
+branches:
+  only:
+  # Since we're using conventional commit paradigm, we only want a master merge commit to trigger a build: https://www.conventionalcommits.org/en/v1.0.0-beta.4/
+  - master
+env:
+  global:
+  # This environment variable is to support the rollover from the initial beta artifact release
+  - BETA_TAG="-beta"
+before_script:
+  # Before updating versioning and pushing artifacts, copy over the maven settings file with ossrh and gpg credentials
+  - cp .travis.settings.xml $HOME/.m2/settings.xml
+script:
+  # Find the most recent version via git tags
+  - OLD_VERSION=$(npx git-semver-tags | head -n 1)
```

Comment:
I'm not completely sure if the image this will run on will have npx or anything like that installed.

Reply:
```
$ npx --version
9.6.0

$ gpg --version
gpg (GnuPG) 1.4.16
```

NPX and GPG are both on the vm